src/components/Post-Components/MakePost.tsx

```diff
@@ -0,0 +1,42 @@
+import { h } from 'preact'
+import { Box, FormControl } from '@mui/material';
+import { Avatar } from '@mui/material';
+import { blue } from '@mui/material/colors';
```

Comment:
You are not using this file. Delete it

Reply:
This file is currently being unused because the plan was to merge parts of this design with Della's implementation of making a post after the part 1 deadline. After the deadline, we'll probably work on joining the designs together. I will gladly delete this file after we do that.

Reply:
Can you save it somewhere else then? I don't think we should push code that it's not being used yet